tests/testthat/test-Sum.R

```diff
@@ -0,0 +1,10 @@
+context("Sum")
+
+test_that("Test fail", {
+  expect_error(Sum("string"))
+  expect_error(Sum(c("1", "3")))
+})
+
+test_that("Test output", {
+  expect_equal(Sum(as.numeric(c("1", "3"))), Sum(c(1, 3)))
```

Comment:
This test is a little convoluted but its fine to keep it here. I suggest writing a test which is a little more in line with what the function returns.
e.g. `Sum(c(1, 3))` will return the value `4` so we may want to write a test which checks this. Also a test which includes `NA`s in the input vector should be written because the removal of `NA`s is the main point of `Sum()`.